server/scripts/seed.ts

```diff
@@ -227,12 +227,14 @@ const createDonorHomepage = () => {
         "diapers distributed every month",
         "visits to our care closet"
     ];
+    const statType: string[] = Object.keys(StatisticType);
```

Comment:
nit: assign `numStats = min(statIcons.length, stateText.length)` instead of hardcoding `numStats = 3`